evap/evaluation/models.py

```diff
@@ -468,6 +468,30 @@ class State(models.IntegerChoices):
         verbose_name=_("wait for grade upload before publishing"), default=True
     )
 
+    @property
+    def has_exam(self):
+        return self.course.evaluations.filter(name_de="Klausur", name_en="Exam").exists()
+
+    @property
+    def get_earliest_legal_exam_date(self):
```

Comment:
Let's remove "get". Is there anything "legal" to our definition here? Otherwise, I would prefer
```suggestion
    def earliest_possible_exam_date(self):
```